packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -197,6 +209,38 @@ export interface SetLoanToken {
   interest?: BigNumber
 }
 
+export interface TokenData {
+  [key: string]: TokenDetail
+}
+
+export interface TokenDetail {
```

Comment:
```suggestion
export interface LoanTokenInfo {
```
follow the naming `TokenInfo`